bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableTableAdminGrpcClient.java

```diff
@@ -96,6 +104,13 @@
   private final BigtableAsyncRpc<CreateTableFromSnapshotRequest, Operation>
       createTableFromSnapshotRpc;
```

Comment:
Shouldnt we remove the old snapshot methods? Or at the very least deprecate them?

Reply:
Will deprecate them for this release.